test/hotspot/gtest/runtime/test_os_linux.cpp

```diff
@@ -200,8 +200,8 @@ TEST_VM(os_linux, reserve_memory_special_huge_tlbfs_size_not_aligned_with_bad_re
       char* p = HugeTlbfsMemory::reserve_memory_special_huge_tlbfs(size, alignment, lp, req_addr, false);
       HugeTlbfsMemory mr(p, size);
       // as the area around req_addr contains already existing mappings, the API should always
-      // return NULL (as per contract, it cannot return another address)
-      EXPECT_TRUE(p == NULL) << " size = " << size
+      // return nullptr (as per contract, it cannot return another address)
```

Comment:
s/nullptr/null/

Reply:
This is technically correct.  Changing the comments like this seems really pointless.